.github/workflows/CI.yml

```diff
@@ -45,15 +45,15 @@ jobs:
 
       # Restoring solution packages
       - name: Restore packages
-        run: dotnet restore
+        run: dotnet restore DotnetStarter.sln
       
       # Building the solution
       - name: Solution build
-        run: dotnet build --configuration Release --no-restore
+        run: dotnet build DotnetStarter.sln --configuration Release --no-restore
 
       # Testing the solution
       - name: Solution test
-        run: dotnet test --configuration Release --no-build --verbosity normal --collect:"XPlat Code Coverage" --results-directory ./coverage
+        run: dotnet test DotnetStarter.sln --configuration Release --no-build --verbosity normal --collect:"XPlat Code Coverage" --results-directory ./coverage
```

Comment:
Do we need these explicit solution names specified?

Reply:
Yes, see the [failing pipeline from earlier](https://github.com/Enterwell/dotnet-starter/actions/runs/7504126099/job/20430421041) 

In short, after adding docker-compose dotnet restore/build/run needs to have a specified project or solution